security/providers/oidc-common/src/main/java/io/helidon/security/providers/oidc/common/OidcConfig.java

```diff
@@ -1458,6 +1580,7 @@ public Builder cookieName(String cookieName) {
          * @param cookieName name of a cookie
          * @return updated builder instance
          */
+        @ConfiguredOption(key = "cookie-name-id-token", value = DEFAULT_ID_COOKIE_NAME)
```

Comment:
The `key` is the one that is "guessed" from the method name, so it should not be defined.